README.md

```diff
@@ -51,6 +51,7 @@ installation and postprocessing procedures. Its features include:
   * Acoustic perturbation equations
   * Hyperbolic diffusion equations for elliptic problems
   * Lattice-Boltzmann equations (D2Q9 and D3Q27 schemes)
+  * Shallow water equations
```

Comment:
Yes, I already discussed with @ranocha and this new equation system is not experimental as it fits into the current Trixi API. It would only introduce breaking changes once curvilinear shock capturing with nonconservative terms #736 is addressed. I totally forgot to update the files you mentioned. I will do so.

Reply:
Done. I have updated the NEWS and index files